### DISCUSSION TITLE
zsh_history

### DISCUSSION THREAD

**Discussion by acsezen** [twpayne/chezmoi#3515] [Q&A] [answered]
Hello,

I am attempting to manage my .zsh_history file using chezmoi, but I'm encountering inconsistent results. Sometimes it successfully creates a symbolic link, while other times it fails. I believe I might be overlooking something important in this process, but I'm not exactly sure what it is.

`% ls -al ~/.local/share/chezmoi/.zsh_history`
`.rw-r--r-- 738k xxx 25 Jan 14:43 /home/xxx/.local/share/chezmoi/.zsh_history`

`% cat ~/.local/share/chezmoi/private_dot_config/private_zsh/symlink_dot_zsh_history.tmpl`
`{{ .chezmoi.sourceDir }}/.zsh_history%`

`% ls -al ~/.config/zsh/.zsh_history`
`.rw-r--r-- 605k xxx 25 Jan 14:48 /home/xxx/.config/zsh/.zsh_history`

I get the idea from https://www.chezmoi.io/user-guide/manage-different-types-of-file/#handle-configuration-files-which-are-externally-modified .

Thanks for the answers!

**Comment by twpayne** [comment, marked as answer]
chezmoi is not suitable for managing shell history. See [this FAQ entry](https://www.chezmoi.io/user-guide/frequently-asked-questions/usage/#can-i-use-chezmoi-to-manage-my-shell-history-across-multiple-machines). Use an appropriate tool like [Atuin](https://atuin.sh/) instead.
